scripts/assets/maven_dependencies.textproto

```diff
@@ -9,17 +9,6 @@ maven_dependency {
     }
   }
 }
-maven_dependency {
-  artifact_name: "androidx.appcompat:appcompat:1.2.0"
-  artifact_version: "1.2.0"
-  license {
-    license_name: "The Apache Software License, Version 2.0"
-    original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
-    scrapable_link {
-      url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
-    }
-  }
-}
 maven_dependency {
```

Comment:
To verify that the CI check is working, I deleted this dependency from the list. And the check failed as expected.

Reply:
Suggest adding screenshot of failing CI with appropriate error.

Reply:
Done.